### PR TITLE
Support Bridge Usdc Noble

### PR DESCRIPTION
**Pull request by haunv3**

### CODE REVIEW

**Comment by tucq88** [comment]
- [ ] USDC Token value not display
image

- [ ] Chart not display with USDC - https://gyazo.com/a48dd0b70ae6036314eb371b49e89520 
- [ ]

**Comment by trungbach** [comment]
> * [ ]  USDC Token value not display
> 
> image
> * [ ]  Chart not display with USDC - https://gyazo.com/a48dd0b70ae6036314eb371b49e89520
> * [ ]

@tucq88 chart chỉ có dữ liệu của những cặp nằm trong list pools của mình đó ạ

File: src/pages/Balance/index.tsx [hunk 1]
```diff
@@ -182,12 +178,20 @@ const Balance: React.FC<BalanceProps> = () => {
       }
       if (newToToken.coinGeckoId !== from.coinGeckoId)
         throw generateError(`From token ${from.coinGeckoId} is different from to token ${newToToken.coinGeckoId}`);
+
+      const latestOraiAddress = await window.Keplr.getKeplrAddr();
+      // has to switch network to the correct chain id on evm since users can swap between network tokens
+      if (!window.Metamask.isTron(from.chainId) && !from.cosmosBased) {
```
Comment (ducphamle2) [inline src/pages/Balance/index.tsx:184]: I think its best if we directly validate if from is evm based. This condition will go wrong if there's another network that is neither tron or cosmos based (like bitcoin)

File: src/pages/Balance/index.tsx [hunk 2]
```diff
@@ -159,15 +159,11 @@ const Balance: React.FC<BalanceProps> = () => {
         processTxResult(from.rpc, result, `${KWT_SCAN}/tx/${result.transactionHash}`);
         return;
       }
-      if (from.cosmosBased) {
+      // TODO: hardcode check case USDC oraichain -> USDC noble
+      if (from.cosmosBased && filterNetwork !== 'noble-1') {
```
Comment (ducphamle2) [inline src/pages/Balance/index.tsx:163]: I still strongly believe that we don't have to hardcode noble-1 here. If theres a technical barrier, pls disclose

**Review by ducphamle2 — CHANGES_REQUESTED** [review]

**Pull request merged**